test/test_vim.py

```diff
@@ -22,7 +22,8 @@ def test_command():
     vim.command('w')
     ok(os.path.isfile(fname))
     eq(open(fname).read(), 'testing\npython\napi\n')
```

Comment:
@brcolow Try changing this so that the `open()`'d file gets closed before we try `os.unlink()` below. http://askubuntu.com/a/701536 Perhaps this:

```python
with open(fname) as f:
    eq(f.read(), 'testing\npython\napi\n')
```

Then we shouldn't need to check `os.name != 'nt'` below.